diff --git a/lib/module/createMMKV.js b/lib/module/createMMKV.js
index 644840da7b021b34deef93e89c8b68bbaf488546..f59d4367885234ca7c5b6c50650aebb2d15082e6 100644
--- a/lib/module/createMMKV.js
+++ b/lib/module/createMMKV.js
@@ -32,7 +32,7 @@ export const createMMKV = config => {
     }
 
     // Check if we are running on-device (JSI)
-    if (global.nativeCallSyncHook == null || MMKVModule.install == null) {
+    if (MMKVModule.install == null) {
       throw new Error('Failed to create a new MMKV instance: React Native is not running on-device. MMKV can only be used when synchronous method invocations (JSI) are possible. If you are using a remote debugger (e.g. Chrome), switch to an on-device debugger (e.g. Flipper) instead.');
     }
 
